packages/@okta/vuepress-site/docs/concepts/scim/index.md

```diff
@@ -9,15 +9,15 @@ meta:
 
 This topic covers the concepts and use cases for SCIM and Okta.
 
-When you’re ready to create a SCIM integration, see [Build a SCIM provisioning integration](/docs/guides/scim-provisioning-integration-overview) and [how to implement the SCIM protocol with Okta](https://developer.okta.com/docs/api/openapi/okta-scim/guides/).
+When you're ready to create a SCIM integration, see [Build a SCIM provisioning integration](/docs/guides/scim-provisioning-integration-overview). For reference information on how to implement SCIM with Okta, see [SCIM Protocol](https://developer.okta.com/docs/api/openapi/okta-scim/guides/).
 
 ## What is SCIM for?
 
 SCIM, or the [System for Cross-domain Identity Management](https://scim.cloud) specification, is an open standard designed to manage user identity information. SCIM provides a defined schema for representing users and groups, and a RESTful API to run CRUD operations on those user and group resources.
 
 The goal of SCIM is to securely automate the exchange of user identity data between your company's cloud apps and any Service Providers (SP).
 
-Managing user lifecycles in your org is a fundamental business problem. Hiring employees is just the first step. You also need to provision apps needed for their job and enforce corporate security policies. Then you need to update user accounts as they advance through your company. At the end of their employment, you need to ensure that all access is quickly and thoroughly revoked across all apps. Handling all of this can be time-consuming and error-prone if done manually.
+Managing user lifecycles in your org is a fundamental business problem. Hiring employees is just step one. You also need to provision apps for their job and enforce corporate security policies. Then you need to update user accounts as they advance through your company. At the end of their employment, you need to ensure that all access is quickly and thoroughly revoked across all apps. Manually handling all of this can be time-consuming and error-prone.
```

Comment:
I think I'd leave this as "the first step" unless we're specifically going to number the remaining ones. I don't see any reference to additional steps in this doc, but maybe they're in a different one?